GDevelop.js/types/gdextensionpropertiesmanager.js

```diff
@@ -0,0 +1,13 @@
+// Automatically generated by GDevelop.js/scripts/generate-types.js
+// Automatically generated by GDevelop.js/scripts/generate-types.js
+// Automatically generated by GDevelop.js/scripts/generate-types.js
+// Automatically generated by GDevelop.js/scripts/generate-types.js
+// Automatically generated by GDevelop.js/scripts/generate-types.js
```

Comment:
That's a lot of comments :D What happened?

Reply:
Idk tbh. Oh wait I think it's because the branch is old and it uses the version of the extractor that was broken on windows.